doc/dev/mountpoints.md

```diff
@@ -0,0 +1,179 @@
+
+To illustrate this structure, here is an example configuration:
+
+```
+# List of plugins with their config
+system:/elektra/mountpoints/\/hosts/plugins/#0/name (="resolver_fm_hpu_b")
+system:/elektra/mountpoints/\/hosts/plugins/#1/name (="glob")
+system:/elektra/mountpoints/\/hosts/plugins/#1/config/set/#0
+system:/elektra/mountpoints/\/hosts/plugins/#1/config/set/#1
+system:/elektra/mountpoints/\/hosts/plugins/#1/config/set/#2
+system:/elektra/mountpoints/\/hosts/plugins/#1/config/set/#3
+system:/elektra/mountpoints/\/hosts/plugins/#1/config/set/#4/flags
+system:/elektra/mountpoints/\/hosts/plugins/#2/name (="hosts")
+system:/elektra/mountpoints/\/hosts/plugins/#3/name (="sync")
+system:/elektra/mountpoints/\/hosts/plugins/#4/name (="error")
+system:/elektra/mountpoints/\/hosts/plugins/#5/name (="network")
+system:/elektra/mountpoints/\/hosts/plugins/#6/name (="backend")
+
+# Define backend plugin
+system:/elektra/mountpoints/\/hosts/backend (="#6")
+
+# Configuration for backend plugin
+system:/elektra/mountpoints/\/hosts/path (="myhosts")
+
+system:/elektra/mountpoints/\/hosts/positions/get/resolver = (="#0")
+system:/elektra/mountpoints/\/hosts/positions/get/storage = (="#2")
+system:/elektra/mountpoints/\/hosts/positions/get/poststorage/#0 (="#1")
+
+system:/elektra/mountpoints/\/hosts/positions/set/resolver = (="#0")
+system:/elektra/mountpoints/\/hosts/positions/set/prestorage/#0 = (="#1")
+system:/elektra/mountpoints/\/hosts/positions/set/prestorage/#1 = (="#4")
+system:/elektra/mountpoints/\/hosts/positions/set/prestorage/#2 = (="#5")
+system:/elektra/mountpoints/\/hosts/positions/set/storage = (="#2")
+system:/elektra/mountpoints/\/hosts/positions/set/precommit/#0 = (="#3")
+system:/elektra/mountpoints/\/hosts/positions/set/commit = (="#0")
+system:/elektra/mountpoints/\/hosts/positions/set/rollback (="#0")
+```
+
+Explanation:
+
+- This configures a mountpoint `/hosts` that loads the plugins `resolver_fm_hpu_b`, `glob`, `hosts`, `sync`, `error`, `network` and `backend`.
+  The plugin `glob` also has some configuration keys defined.
+- It then selects the `backend` plugin as the backend plugin.
+- The rest is the mountpoint definition specific to the `backend` plugin.
+
+As you can see from this example, the classic "positions" are part of the plugin-specific mountpoint definition.
+This allows different backend plugins to use different "positions".
+
+For example, imagine a `version` backend plugin that always just provides a few keys with version information for Elektra.
+It doesn't need to call any other plugins and requires no configuration at all.
+A mountpoint with this plugin could look like this:
+
+```
+system:/elektra/mountpoints/\/version/plugins/#0/name (="version")
+system:/elektra/mountpoints/\/version/backend (="#0")
+```
+
+## Operations and Phases
+
+See [KDB Operations Documentation](./kdb-operations.md) for a description of operations and phases.
```

Comment:
Why we have this section here? I would simply link at it in `See Also` at end.

Reply:
I think I originally started to write the whole operations description here and then move to another file. A simple link would probably be enough. Especially, if we add an overview document that links to the various pieces and tells you in what order you should read them.